.github/workflows/support-frontend-build.yml

```
@@ -0,0 +1,62 @@
+name: Build support-frontend
+
+on:
+  pull_request:
+    branches: [ main ]
+
+env:
+  GU_SUPPORT_WORKERS_LOAD_S3_CONFIG: false
+
+jobs:
+  support_frontend_build:
+    if: github.actor != 'scala-steward'
```

Doing some experiments in guardian/payment-failure-comms#95 where facing the same problem, this seems to work:
```suggestion
    if: github.event.pull_request.head.repo.owner.login == 'guardian'
```

---

Thanks, that's useful